docs/user-guide/rest_endpoints.md

```diff
@@ -0,0 +1,34 @@
+# Measure endpoint (/evaluation)
+## Description
+This endpoint will take a patient and a measure and return the measure report for that patient evaluated against that measure
+## Parameters
+
+- patientId: A single patient intended to be run.
+- measureId: The id of the measure to run. You do not need a measureId and an identifier, you only need one or the other.
+- identifier: The identifier of the measure to run. If you provide this, you do not need a measureId but you do need a version.
+- version: The version of the measure to run. This is only necessary if you're locating your measure with the identifier rather than the measureId.
+- parameters: A place to specify any neccessary input parameters for your define. For example, if your define requires a measurement period this is where you would supply it.
+- evidenceOptions: Two options for adding extra information to the returned measure report.
+	- includeEvaluatedResources: Whether or not to include the resources that were evaluated to determine if patients succeed or fail in the measure report.
+	- defineReturnOption: If this is set to ALL, will return add the define result to the report regaredless of the type. If set to BOOLEAN and the define returns a boolean type, the boolean will be added to the measure report. If it is set to BOOLEAN and the define return type is not BOOLEAN, nothing will be added to the measure report.
```

Comment:
When we discussed as a team, I believe the agreement was that we weren't going to duplicate the documentation that is already stated in the Swagger in the user guide. The intention was to provide additional, detailed behavior on API behavior that wasn't REST specific. Mainly, I had an interest in creating a section (either in the MeasureEvaluator section of the getting started guide or a completely new page) that detailed the evidence options because those are relevant whether you are a Java API or REST API consumer. I really think that everything related to quality measure evaluation could/should be a separate page in the user guide.

For "includeEvaluatedResources":
1) I'd like it to be clear about _where_ the evaluated resources are included and how. Specifically, the MeasureReport resource's [evaluatedResources ](https://www.hl7.org/fhir/measurereport-definitions.html#MeasureReport.evaluatedResource) property is populated when this flag is set to true. It is provided as an option because the data for that flag can get quite large and may not be necessary for the consuming application. We can drive storage and processing efficiencies by leaving them out.
2) Need to be clear about what the default value is here.

For "defineReturnOption": 
1) you've got a grammatical error near "return add". Probably choose one or the other verb. 
2) I think we should expand this section to be absolutely clear about how the data is rendered in the MeasureReport. I would mention the CDM IG extension by URI, show an example, and specifically call out that define statements that return whole resources are rendered as references to the resource and not the resource itself.
3) I think we should call out that this option is intended for users that need to do debugging of the CQL that is executed in support of the measure evaluation and that it defaults to NONE.

Reply:
Ok, I had gotten exactly the opposite out of that conversation, but I can move these and delete this page.